ui/src/vayla-design-lib/closeable-modal/closeable-modal.tsx

```diff
@@ -9,25 +9,23 @@ type CloseableModalProps = {
     offsetY?: number;
     children: React.ReactNode;
     className?: string;
+    useRefWidth?: boolean;
+    offsetWidth?: number;
 };
 
 export const CloseableModal: React.FC<CloseableModalProps> = ({
     positionRef,
     onClickOutside,
-    offsetX = 0,
-    offsetY = 0,
     children,
     className,
+    offsetX = 0,
+    offsetY = 0,
+    offsetWidth = 0,
+    useRefWidth = false,
```

Comment:
Tää on mun silmiinä vähän jännä, kun kumminkaan `width` ei oo aina ihan täsmälleen sidottu refin leveyteen (koska sille voi antaa myös offsetin.) Tälle voisi ehkä keksiä jonkun vähän kuvaavamman nimen, joskaan en itse keksi nopeasti parempiakaan ideoita. Ehkä joku `baseOnRefWidth` tms.? Huono tosin sekin)

Reply:
offsetWidth ja useRefWidth on naimisissa keskenään, mitenkä offsetRefWidth?

Reply:
Se voisi olla. Tosin ehkä sitten toisin päin käännettynä, eli `refWidthOffset`